Gemfile.lock

```diff
@@ -70,14 +95,54 @@ GEM
     rb-fsevent (0.11.2)
     rb-inotify (0.11.1)
       ffi (~> 1.0)
-    rexml (3.3.4)
-      strscan
+    rexml (3.3.7)
```

Comment:
Does this mean we don't need to merge https://github.com/jamulussoftware/jamuluswebsite/pull/1008 ?

Reply:
That's correct. When this has been merged, dependabot should close that PR. If it doesn't do so automatically, it can be triggered with a dependabot rebase comment.

Reply:
Ah, #1008 is raised against `release`. This PR is raised against `next-release`. My understanding is when the `next-release` branch gets moved/merged to `release`, #1008 won't be needed any more.

Reply:
Yes. @softins is correct.